libmate-panel-applet/mate-panel-applet.h

```diff
@@ -107,7 +111,9 @@ void mate_panel_applet_set_size_hints(MatePanelApplet* applet, const int* size_h
 
 gboolean mate_panel_applet_get_locked_down(MatePanelApplet* applet);
 
+#ifdef HAVE_X11
```

Comment:
This is public file that will be used by 3rd party applets. Do you expect all applet developers to define this in their own `config.h` files?

Reply:
Ifdef removed